apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/FissClaimResponseTransformerV2.java

```diff
@@ -145,7 +147,9 @@ private static List<Identifier> getIdentifier(PreAdjFissClaim claimGroup) {
                         C4BBIdentifierType.UC.getSystem(),
                         C4BBIdentifierType.UC.toCode(),
                         C4BBIdentifierType.UC.getDisplay())))
-            .setSystem(BBCodingSystems.FI_DOC_CLM_CONTROL_NUM)
+            .setSystem(
+                CCWUtils.calculateVariableReferenceUrl(
+                    CcwCodebookMissingVariable.FI_DOC_CLM_CNTL_NUM))
```

Comment:
This is changing from "/variables/fi_doc_clm_cntrl_num" to "/variables/fi_doc_clm_cntl_num" right? Is this a breaking change, given that the system is essentially changing here?

Reply:
It would be, if the PACA resources (Claim/ClaimResponse) were in production, but we're not there yet.

Reply:
Ah ok so this is not released yet, all good then.